keps/2076-kueuectl/README.md

```diff
@@ -330,6 +331,16 @@ kueuectl resume workload name --namespace=ns
 Output:
 None.
 
+### Delete Workload
```

Comment:
Adding context and motivations for dedicated commands would be helpful to understand this command objective.

Reply:
Done